pkg/kcp/provider/aws/meta/metadata.go

```diff
@@ -115,3 +128,10 @@ const (
 	ElastiCache_CREATE_FAILED ElastiCacheState = "create-failed"
 	ElastiCache_SNAPSHOTTING  ElastiCacheState = "snapshotting"
 )
+
+func AnyConditionChanged(obj composed.ObjWithConditions, conditionsToSet ...metav1.Condition) bool {
```

Comment:
What this has to do with aws metadata? Ideally this should be part of the `composed.(Patch|Update)Status` but at least move it out of aws/meta and put somewhere else, maybe in `composed`.

Reply:
Yes, I thought about that as well that ideally this should be integrated `into composed.(Patch|Update)Status` but for now I would just move it to `composed`.